### PR TITLE
completed task 7 and ran some test

### PR DESCRIPTION
**Pull request by CanAn173**

### CODE REVIEW

File: __tests__/creatingApp.test.js [hunk 1]
```diff
@@ -125,6 +125,102 @@ describe("/api", () => {
...
+        expect(receivedComment).toMatchObject(
+          {
+            msg: 'Bad Request'
+          }
+        )
+      })
+    })
   });
 
+
```
Comment (Beth-McCormack) [inline __tests__/creatingApp.test.js:242]: Could you add tests for 
* missing required fields eg no username or no body
* if the username doesnt exist

File: __tests__/creatingApp.test.js [hunk 2]
```diff
@@ -125,6 +125,102 @@ describe("/api", () => {
...
+          body: 'I enjoy coding?',
+          review_id: 3,
+          created_at: expect.any(String)
+        }
+      )
+
+      });
+    });
+
+    test("404-POST-api/reviews/:review_id/comments", () => {
```
Comment (Beth-McCormack) [inline __tests__/creatingApp.test.js]: Please change this description to say what the test is testing for eg which status code and why, like above

File: __tests__/creatingApp.test.js [hunk 3]
```diff
@@ -125,6 +125,102 @@ describe("/api", () => {
...
+        expect(receivedComment).toMatchObject(
+          {
+            msg: 'Not found'
+          }
+        )
+      })
+      
+    });
+
+    test("400-POST-api-reviews/:review_id/comments", () => {
```
Comment (Beth-McCormack) [inline __tests__/creatingApp.test.js]: see above comment

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comments

**Review by emilyb93 — APPROVED** [review]

**Pull request merged**